### DISCUSSION TITLE
Unable to upload files with formdata multi-part

### DISCUSSION THREAD

**Discussion by AhmedMustafa505** [microsoft/react-native-windows#12167] [Q&A] [answered]
I'm implementing a react-native-windows application, and trying to upload file to our server but every time I get Network Error, however the same code works fine on Android.

P.s: I tried to make an http request for a global api, and I succeeded. so the issue is relying behind the form-data technique or RNW configurations I'm missing. 

`const uploadFile = async () => {
    const pickerResult = await DocumentPicker.pickSingle({
      presentationStyle: 'fullScreen',
      copyTo: 'documentDirectory',
      type: [DocumentPicker.types.allFiles],
      mode: 'import',
    });
    console.log(pickerResult);

    const response = uploadImage(pickerResult);
  };
  const uploadImage = async file => {
    const formData = new FormData();
    formData.append('file', {
      name: file.name,
      type: file.type, 
      uri: file.uri, 
    });

    console.log(formData);
    try {
      const response = await axios.post(
        'http://xxx.xxx.xx.xx:xxxx/file/upload',
        formData,
        {
          headers: {
            'Content-Type': 'multipart/form-data',
            Accept: 'application/json',
          },
          timeout: 5000,
        },
      );

      console.log('Image upload success:', response.data);
    } catch (error) {
      console.error('Image upload error:', error);
    }
  };`

**Comment by chrisglein** [comment, marked as answer]
Looks like you've opened an issue to track this. We'll continue any further discussion there: https://github.com/microsoft/react-native-windows/issues/12168
